pkg/results/report.go

```diff
@@ -146,9 +146,63 @@ func (r *reporter) report(request Request) {
 		logrus.Tracef("could not create report request: %v", err)
 		return
 	}
+	sendRequest(req, r.client, r.username, r.password)
+}
+
+type PodScalerRequest struct {
```

Comment:
you need to make sure to remove the `podScalerRequest` from `result-aggregator` and use this one there too.